atlasdb-api/src/main/java/com/palantir/atlasdb/spi/KeyValueServiceConfigHelper.java

```diff
@@ -26,4 +26,9 @@
     default Optional<String> namespace() {
         return Optional.empty();
     }
+
+    @Override
+    default int maxConcurrentGetRanges() {
+        return 4;
```

Comment:
it might be nice if in the executor code we were able to allow an unlimited queue but still provide a warning for people who are very underwater on this setting (e.g. "WARN: You have 3000 pending getRanges tasks; consider the choices you've made in your life up till now and additionally consider changing maxConcurrentGetRanges")